resolwe_bio/kb/serializers.py

```diff
@@ -19,3 +19,4 @@ class Meta:
         """Serializer configuration."""
 
         model = Feature
+        fields = '__all__'
```

Comment:
This doesn't change any behaviour. It just needs to be specified explicitly in new version of DRF.